src/main/java/world/bentobox/bentobox/managers/AddonsManager.java

```diff
@@ -168,8 +168,11 @@ private void handleAddonIncompatibility(Addon addon) {
     private void handleAddonError(Addon addon, Throwable throwable) {
         // Set the AddonState as "ERROR".
         addon.setState(Addon.State.ERROR);
-        plugin.log("Skipping " + addon.getDescription().getName() + " due to an unhandled exception...");
-        plugin.log("STACKTRACE: " + throwable.getClass().getSimpleName() + " - " + throwable.getMessage() + " - " + throwable.getCause());
+        plugin.logError("Skipping " + addon.getDescription().getName() + " due to an unhandled exception...");
+        plugin.logError("STACKTRACE: " + throwable.getClass().getSimpleName() + " - " + throwable.getMessage() + " - " + throwable.getCause());
+        if (plugin.getConfig().getBoolean("debug")) {
```

Comment:
This is a "secret" debug setting to enable us to look at the stack trace if it occurs. Otherwise there's no way to see what's going on without recompiling BentoBox.

Reply:
Okay, that's fine. We might want to add a "nicer" debug tool somewhere at some point.